src/main/java/uk/gov/pay/api/ledger/service/LedgerUriGenerator.java

```diff
@@ -18,6 +18,7 @@ public LedgerUriGenerator(PublicApiConfig configuration) {
     }
 
     public String transactionsURIWithParams(Map<String, String> queryParams) {
+        queryParams.put("status_version", "1");
```

Comment:
how about documenting the meaning of this using an enum?

Reply:
Could potentially be candidate for ledger docs ? (for which we don't have API docs at the moment)